benchmarks/Kestrel.Performance/Kestrel.Performance.csproj

```diff
@@ -4,7 +4,7 @@
     <!-- Using shorter assembly name instead of Microsoft.AspNetCore.Server.Kestrel.Performance because https://github.com/dotnet/BenchmarkDotNet/issues/498 -->
     <AssemblyName>Kestrel.Performance</AssemblyName>
     <RootNamespace>Microsoft.AspNetCore.Server.Kestrel.Performance</RootNamespace>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
@sebastienros @davidfowl was there a reason these weren't updated? Something about running the benchmarks on old framework infrastructure?